src/site/content/en/blog/learn-to-bundle/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Learn to bundle
+subhead: |
+    Learn to bundle your website.
+date: 2019-11-11
+hero: hero.png
+alt: A logo of a web bundle
+authors:
+  - kenjibaheux
```

Comment:
Could you please add my self as an author? The meta data we were planning is listed in the [doc](https://docs.google.com/document/d/1NFStsELbOiW5ajKWVKSGlj-sYuF6YbLb0Iq8zUl_xHQ/edit) but 
- having @uskay and @KenjiBaheux as an author
- Fixing the URL to /web-bundles (doesn't the directory have to be /web-bundles/ in this case?)
would be great.